src/SCREENS.js

```diff
@@ -4,6 +4,7 @@
  */
 export default {
     HOME: 'Home',
+    THIRD_PARTY_SIGN_IN: 'third-party-sign-in',
```

Comment:
We don't want to commit this. We just want the component.

The apple and google screens will have different routes, so we don't want to share this, and generally we've been avoiding the "third-party sign-in" language where possible.